### DISCUSSION TITLE
Can LogicalName be added to FSharpParameter?

### DISCUSSION THREAD

**Discussion by En3Tho** [dotnet/fsharp#11255] [Ideas]
Could be used to enable this (note the highlightings): 
![image](https://user-images.githubusercontent.com/37334640/111486861-e7209780-8748-11eb-8d32-e5c3f383e689.png)

over this:
![image](https://user-images.githubusercontent.com/37334640/111486636-b50f3580-8748-11eb-9081-7e0d14b89ba7.png)

screenshots are from Rider but in VS it's the same
@cartermp

**Comment by cartermp** [comment]
Yeah, I don't see any issue doing that.
